src/resources/Sources/SourcesInterfaces.ts

```diff
@@ -257,6 +278,31 @@ export interface FormAuthenticationInput {
     value?: string;
 }
 
+export interface FormAuthenticationConfig {
+    actionMethod?: string;
+    actionUrl?: string;
+    addFoundInputs?: boolean;
+    authenticationFailed?: FormAuthenticationFailedConfiguration;
+    enableJavaScript?: boolean;
+    formContentType?: string;
+    formId?: string;
+    formUrl?: string;
+    inputs?: FormInput[];
+    javaScriptLoadingDelayInMilliseconds?: number;
+    customLoginSequence?: JSON;

```

Comment:
`JSON` is not a type, its only the object containing the conversion methods `stringify` and `parse`

Reply:
It should either be `string` (if the JSON is stringified) or `any` (if the JSON is parsed)